### ISSUE TITLE
Font weight is permanently bold after latest update

### ISSUE THREAD

**Issue by alexmanzo** [microsoft/vscode#225271]
Does this issue occur when all extensions are disabled?: No

- VS Code Version: 1.92.1
- OS Version: MacOS Sonoma 14.6

Steps to Reproduce:
This looks to be a regression of this issue: https://github.com/microsoft/vscode/issues/214390
I'm using `Operator Mono` font.
1. Open editor with affected font, verify font weight is 400. Font shows up as bold.

Workaround is launching from cli with `--disable-blink-features=FontMatchingCTMigration`

**Comment by SimonGodefroid** [comment]
Cheers for re-opening. Was losing my mind after seeing it had been fixed in 1.90.2 but reproducing in 1.92.0 today. 
Pasting the full opening command for mac users.

```bash
open -a "Visual Studio Code" --args --disable-blink-features=FontMatchingCTMigration
```

**Comment by rzhao271** [comment]
Duplicate of https://github.com/microsoft/vscode/issues/224496
